ipapython/certdb.py

```diff
@@ -749,7 +750,8 @@ def import_files(self, files, import_keys=False, key_password=None,
 
                     continue
 
-            raise RuntimeError("Failed to load %s" % filename)
+            raise RuntimeError(
+                "Failed to load %s, please check key password" % filename)
```

Comment:
It seems that we can reach here when:
- the file is a PKCS #12 file but the password is wrong, or
- the file was not a valid PKCS #12 (or any of the preceding formats that were tried).

Ideally we should distinguish these cases.  It would not be helpful to suggest that the problem might be an invalid passphrase, when in fact the file is an unrecognised format.  `import_pkcs12` internals already distinguish the invalid password case from other cases (based on `pk12util` exit status).  So, would it be better to allow the exceptions thrown by `import_pkcs12` to bubble up, instead of suppressing it (line 727)?  The messages of the exceptions thrown by `import_pkcs12` can be tweaked for consistency with the other errors.

Reply:
Thanks for your comments, I had missed that point.